skaffold.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: skaffold/v1
+kind: Config
+build:
+  artifacts:
+    - image: lab-partner
+      context: ./
+  local: {}
+
+deploy:
+  helm:
+    releases:
+      - name: lab-partner
+        chartPath: charts/lab-partner
+        wait: true
+        values:
+          image.name: lab-partner
+        setValues:
+          ingress.host: localhost
```

Comment:
this shouldn't be the default for all skaffold deployments.  you can add a profile for your local cluster based on the kube context to set this locally if you want.